libraries/SD/src/SD.h

```diff
@@ -32,6 +33,8 @@ class SDFS : public FS
     void end();
     sdcard_type_t cardType();
     uint64_t cardSize();
+    uint64_t totalBytes(const char * logical_drive="0:", FRESULT * error=NULL);
+    uint64_t usedBytes(const char * logical_drive="0:", FRESULT * error=NULL);
```

Comment:
Please forward declare ```FRESULT``` and not expose ```ff.h``` here :)

Reply:
Ok I can add 
```
enum FRESULT : unsigned int;
```
instead but the compilation failed because FRESULT is not explicitly typed as unsigned int ( it failed also if no type is provided)
Sorry I am not familiar with enum declaration forwarding ....


Also in SD_test.ino I then need to add `#include "ff.h"`
because of `FR_OK` and `FILINFO fno;`